server/src/test/java/io/crate/expression/reference/sys/check/node/SysNodeChecksTest.java

```diff
@@ -230,7 +230,7 @@ public void testRecoveryAfterNodesCheckWithCorrectSetting() {
 
         var settings = Settings.builder()
             .put(GatewayService.RECOVER_AFTER_DATA_NODES_SETTING.getKey(), 2)
-            .put(GatewayService.EXPECTED_DATA_NODES_SETTING.getKey(), 2)
+            .put(GatewayService.EXPECTED_DATA_NODES_SETTING.getKey(), 3)
```

Comment:
How is this change relevant to the documentation change? If not I prefer to remove it.

Reply:
My thinking here was that I was trying to clarify that it was legit to bring the cluster online with a minority of data nodes offline, so with the change we would now be testing if cluster checks fail on a scenario with 3 data nodes expected but recover_after_data_nodes set to 2

Reply:
Ah ok, fine. Thanks for the clarification.